### PR TITLE
Quests/cli

### LINKED ISSUE #1566: Quests: Create Functions to work with CLI Output
Currently, the Quests module and Quest features have no way of being displayed in the actual Chiventure game through CLI. To remedy this issue, we need to work with the CLI and Action Management Teams. After meeting and discussing this issue with their teams, we determined which functions were necessary, and now implementation and testing are the only steps remaining.

### PR DESCRIPTION
**Pull request by elizabethli31**
We wanted to have quests displayable through the command line, so we worked with action management and cli for this. They needed us to write functions that stored what we wanted displayed into a string, so we wrote functions that would display all the quests a player has, the task tree associated with a quest, and all the elements associated with a particular task. We created a new file for this (quests_cli) and tested using print statements in pre-existing tests. Once this PR goes through, action management and cli will be able to open theirs which both depend on our functions.

### CODE REVIEW

**Review by Castel-Jaloux — APPROVED** [review]
Everything looks good, should be ready for merging. Nice comments above the helper functions explaining elements of the methodology!

File: include/game-state/player.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ typedef enum completion_status {
 */
 typedef struct player_quest {
     char *quest_id;
-    int completion;
+    completion_status_t completion;
```
Comment (philipchenaux) [inline include/game-state/player.h:30]: Well done changing this to a completion_status_t struct, but please change the documentation of the struct right above.

**Review by philipchenaux — CHANGES_REQUESTED** [review]
One minor stylistic comment, but besides that looks good to go!

**Review by philipchenaux — APPROVED** [review]
Looks good to merge!

**Comment by philipchenaux** [comment]
Issue Score: Excellent

Comments:
Excellent job on this PR, note that next time having a more descriptive title that explains your goal would be more useful.

**Pull request merged**